specification/trafficmanager/resource-manager/Microsoft.Network/2017-09-01-preview/trafficmanager.json

```diff
@@ -0,0 +1,1446 @@
+        ],
+        "operationId": "GeographicHierarchies_GetDefault",
+        "description": "Gets the default Geographic Hierarchy used by the Geographic traffic routing method.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The default Geographic Hierarchy.",
+            "schema": {
+              "$ref": "#/definitions/TrafficManagerGeographicHierarchy"
+            }
+          },
+          "default": {
+            "description": "Default response. It will be deserialized as per the Error definition.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "GeographicHierarchy-GET-default": {
+            "$ref": "./examples/GeographicHierarchy-GET-default.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/trafficmanagerprofiles/{profileName}/heatMaps/{heatMapType}": {
+      "get": {
+        "tags": [
+          "HeatMaps"
+        ],
+        "operationId": "HeatMap_Get",
+        "description": "Gets latest heatmap for Traffic Manager profile.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group containing the Traffic Manager endpoint."
+          },
+          {
+            "name": "profileName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Traffic Manager profile."
+          },
+          {
+            "name": "heatMapType",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "enum": [ "latencyVolumeByLocation" ],
+            "description": "The type of HeatMap for the Traffic Manager profile."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The Traffic Manager heatmap.",
+            "schema": {
+              "$ref": "#/definitions/HeatMapModel"
+            }
+          },
+          "default": {
+            "description": "Default response. It will be deserialized as per the Error definition.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "HeatMap-GET": {
+            "$ref": "./examples/HeatMap-GET.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Network/trafficManagerRealUserMetricsKey": {
```

Comment:
The API signature is not correct for a GET or a PUT. it needs a segment for the resource name.